theme-orange/src/main/java/com/orange/ods/theme/orange/guideline/OrangeGuidelineColors.kt

```diff
@@ -64,12 +64,12 @@ val OrangeGuidelineColors = listOf(
         R.attr.functionalPositive
     ) { it.functional.positive },
     GuidelineColor(
-        OdsColors::error.name,
+        OdsFunctionalColors::negative.name,
         GuidelineColorType.Functional,
         "Negative 200",
         "Negative 100",
         com.google.android.material.R.attr.colorError
```

Comment:
I suppose this should be `R.attr.functionalNegative`, but I don't know why this value does not even exist.